blueocean-dashboard/src/main/js/creation/credentials/CredentialsApi.js

```diff
@@ -1,5 +1,14 @@
 import { capabilityAugmenter, Fetch, UrlConfig, Utils } from '@jenkins-cd/blueocean-core-js';
 
+const DOMAIN = 'blueocean-git-domain';
+const SCOPE = 'USER';
+
+function getCredentialsUrl() {
+    const path = UrlConfig.getJenkinsRootURL();
+    return Utils.cleanSlashes(`${path}/blue/rest/organizations/jenkins/credentials/user/`);
```

Comment:
Good catch. We actually handle this somewhat inconsistently in the app and it should be fixed. I will incorporate these changes.